src/commands/database/mysql/user/create.tsx

```diff
@@ -0,0 +1,146 @@
+import { ExecRenderBaseCommand } from "../../../../lib/basecommands/ExecRenderBaseCommand.js";
+import {
+  makeProcessRenderer,
+  processFlags,
+} from "../../../../rendering/process/process_flags.js";
+import { ReactNode } from "react";
+import { Flags, Args } from "@oclif/core";
+import { assertStatus } from "@mittwald/api-client-commons";
+import { Success } from "../../../../rendering/react/components/Success.js";
+import { Value } from "../../../../rendering/react/components/Value.js";
+
+type Result = {
+  mysqlUserId: string;
+};
+
+export class Create extends ExecRenderBaseCommand<typeof Create, Result> {
+  static description = "Create a new mysql user";
+  static args = {
+    "database-id": Args.string({
+      description: "ID of the MySQL Database to create a user for",
+    }),
+  };
+  static flags = {
+    ...processFlags,
+    "database-id": Flags.string({
+      required: true,
+      description: "ID of the MySQL Database to create a user for",
+    }),
+    "access-level": Flags.string({
+      required: true,
+      description: "Access level for this mysql user",
+      options: ["readonly", "full"],
+    }),
+    description: Flags.string({
+      required: true,
+      description: "Description of the mysql user",
+    }),
+    password: Flags.string({
+      required: true,
+      description: "Password used for authentication",
+      exactlyOne: ["public-key", "password"],
+    }),
+    "access-ip-mask": Flags.string({
+      description: "IP from wich external access will be exclusively allowed",
+    }),
+    "external-access": Flags.boolean({
+      description: "Enable/Disable external access for this user.",
+    }),
+  };
+
+  protected async exec(): Promise<Result> {
+    const process = makeProcessRenderer(
+      this.flags,
+      "Creating a new mysql User",
+    );
+
+    // TODO: implement withMySQLId
+
+    let mysqlDatabaseId: string = "";
+    if (this.flags["database-id"]) {
+      mysqlDatabaseId = this.flags["database-id"];
+    } else if (this.args["database-id"]) {
+      mysqlDatabaseId = this.args["database-id"];
+    } else {
+      await process.error(
+        "Please provide a backup schedule id as flag or argument",
+      );
+    }
```

Comment:
What should `withMySQLId` do in this context? Setting a MySQL database id in the CLI context is currently not supported anyway (and I wouldn't really deem it necessary), and the lookup from short IDs to UUIDs is supported in the API now, anyway.

Reply:
I was under the impression, the with[entity]Id functions where there to handle the handover of the ID via args or flags, just to make usage more convinient. That's also what I mimicked with my implementation. I will revise this and stick to the flag.